sys-fs/mount-zip/mount-zip-1.0.9.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="FUSE file system for ZIP archives"
+HOMEPAGE="https://github.com/google/mount-zip"
+SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
Have you tested the ebuild on both arm platforms?

I would suggest to remove the arm keywords and fill a keywording request for them instead.

Reply:
> Have you tested the ebuild on both arm platforms?

Yes.

I work on ChromeOS, which uses the same `ebuild`-based build system as Gentoo. ChromeOS builds and uses `mount-zip` on all its supported architectures, including AMD-64, ARM-64 and ARM-32. I have personally tested `mount-zip` on all these architectures. Moreover, we run automated tests exercising `mount-zip` on all these architectures as well. Finally, ChromeOS has a comprehensive crash reporting system. I looked at the crash reports and the different usage metrics, and I am confident that `mount-zip` works correctly and reliably on all these architectures. And it has been working for several years.